arduino/builder/compilation.go

```diff
@@ -133,21 +134,23 @@ func (b *Builder) compileFileWithRecipe(
 		b.compilationDatabase.Add(source, command)
 	}
 	if !objIsUpToDate && !b.onlyUpdateCompilationDatabase {
+		commandStdout, commandStderr := &bytes.Buffer{}, &bytes.Buffer{}
+		command.RedirectStdoutTo(commandStdout)
+		command.RedirectStderrTo(commandStderr)
+
+		if b.logger.Verbose() {
+			b.logger.Info(utils.PrintableCommand(command.GetArgs()))
+		}
```

Comment:
Maybe we could factor this in the `PrepareCommandForRecitpe`?

Reply:
Not really, because we have some use cases where we don't want to print that.